src/Gui/DlgSettingsWorkbenchesImp.cpp

```diff
@@ -429,6 +429,7 @@ QStringList DlgSettingsWorkbenchesImp::getDisabledWorkbenches()
 
     if (disabled_wbs_list.isEmpty()) {
         disabled_wbs_list.append(QString::fromLatin1("NoneWorkbench"));
+        disabled_wbs_list.append(QString::fromLatin1("TestWorkbench"));
```

Comment:
I don't think that everybody is happy with this decision. People who need the Test wb won't expect to explicitly enable it first.

Also from a strict point of view it violates the architectural layer pattern. This means a layer is not allowed to access a layer above. Although it doesn't access a function or class but just a name it's still a soft violation.

Reply:
> I don't think that everybody is happy with this decision. 

That can be said of every decision ever 😆 -- I use Test every day, but I think it's OK if I have to manually enable it. We have a LONG list of WBs now, it would be good to make the default enabled set a little leaner to help give some usage guidance to new users. Experienced users can find Preferences pretty easily.

Reply:
Having said that the test workbench could also be disabled by the start-up wizard. So we can remove this when we have the setup wizard.